go.mod

```diff
@@ -19,9 +19,9 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.3.1-0.20210225144425-98d03fe6e9bd
+	github.com/keep-network/keep-common v1.3.1-0.20210226151147-28633644e7e5
 	github.com/keep-network/keep-core v1.3.2-0.20210225151037-3f9f104b3d8b
-	github.com/keep-network/tbtc v1.1.1-0.20210224153824-cd94a765339f
+	github.com/keep-network/tbtc v1.1.1-0.20210303113031-adf361fd085b
```

Comment:
Obviously not a blocker, but should we be tagging a discrete version that includes the Celo artifacts (e.g. v1.1.2)?

Reply:
It is changing quite often recently :grimacing: I think we should tag against a discrete version once we are done with the largest chunks of work.